db/schema.rb

```diff
@@ -15,6 +15,16 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "accessories", force: :cascade do |t|
+    t.string "title"
+    t.text "description"
+    t.decimal "price", precision: 9, scale: 2
```

Comment:
What does the precision and scale do in this context?

Reply:
The precision denotes how many characters the number can be, the scale denotes how many characters can be on the right of the decimal point. From what I found online it is the best way to setup your db to handle a column that will be a currency.